tests/functional/test_check.py

```diff
@@ -292,3 +292,22 @@ def test_check_include_work_dir_pkg(script):
     )
     assert matches_expected_lines(result.stdout, expected_lines)
     assert result.returncode == 1
+
+
+def test_check_integrity_errors_on_missing_files(data, script, tmpdir):
```

Comment:
When it is known that test isn't passing, it's common to mark it as xfail per https://pganssle-talks.github.io/xfail-lightning/:
```suggestion
@pytest.mark.xfail(
    reason='Not yet implemented: https://github.com/pypa/pip/pull/8633',
    strict=True,
)
def test_check_integrity_errors_on_missing_files(data, script, tmpdir):
```

Reply:
Makes sense